### ISSUE TITLE
Use Workshop hub for HS 

### ISSUE THREAD

**Issue by ericvd-ucb** [berkeley-dsep-infra/datahub#1747]
Hi - would it be OK to use the workshop Hub on a trial basis this fall for a Demo with HS students?
These would be Cal students teaching HS students so its still Berkeley related

**Comment by yuvipanda** [comment]
@ericvd-ucb go for it. buds-2020 is also up and we can keep that running too if that's better.

**Comment by yuvipanda** [comment]
@ericvd-ucb in fact, I'd prefer using buds-2020 instead of this. We can setup auth in some form that works. What do you think?

**Comment by ericvd-ucb** [comment]
Sure that's fine. Need is not urgent but I would like to have a HS facing and auth-able hub if we get some projects

**Comment by yuvipanda** [comment]
@chrispyles what do you think of me renaming buds-2020 to something like highschools.datahub.berkeley.edu or something like that we can use for this purpose?

**Comment by chrispyles** [comment]
SGTM

**Comment by yuvipanda** [comment]
Alright, so let's rename buds-2020.datahub.berkeley.edu to highschool.datahub.berkeley.edu.

- [ ] Rename directory here
- [ ] Rename references in CI
- [ ] Get new DNS entries
- [ ] Update the Google Auth entries to point to new domain ([prod credentials](https://console.cloud.google.com/apis/credentials/oauthclient/639319454093-kdi145hid2l842ogtuloedcrnm7me3nn.apps.googleusercontent.com?project=ucb-datahub-2018), [staging credentials](https://console.cloud.google.com/apis/credentials/oauthclient/639319454093-qq13h4lkgt1h6oent2ttmatpju067cu6.apps.googleusercontent.com?project=ucb-datahub-2018))

For auth, there are two ways to go:

- Allow an entire domain of Google Accounts. This is [what we do now](https://github.com/berkeley-dsep-infra/datahub/blob/staging/deployments/buds-2020/config/common.yaml#L23) - emails from @berkeley.edu and @achiveprogram.org are allowed. We can add to this list
- Add individual users' email addresses. I think we can have admins just add these from the admin interface, without having to go via the GItHub process. This should make things much faster, and remove us from the loop.

**Comment by felder** [comment]
@yuvipanda is this documentation correct and current?

https://github.com/berkeley-dsep-infra/datahub/blob/staging/docs/admins/howto/new-hub.rst

According to that I figured I should be able to simply change the naming of buds-2020 everywhere in the source where it appears as well as make provisions for that change in the nfsserver, dns, and authentication.  However the first error I got was a nonexistent namespace which I tried to correct with "kubectl create namespace highschool-staging"  That allowed the deploy to seemingly progress, but now it simply times out.
